packages/story-editor/src/utils/cleanForSlug.js

```diff
@@ -15,18 +15,23 @@
  */
```

Comment:
Wonder if this function should live in the url package?

Reply:
Don't see a need right now, since it's editor-specific. But we can discuss in a separate ticket.